dotnetcore/NetstandardModuleSample/src/NetstandardModuleSample/module_dev_sample.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+  "loaders": [
+    {
+      "type": "dotnetcore",
+      "name": "dotnetcore",
+      "configuration": {
+        "binding.path": "dotnetcore.dll",
+        "binding.coreclrpath": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\coreclr.dll",
```

Comment:
What if I am a Linux the developer ? Can you make sure that if the GW will seek for loader path automatically ?